### DISCUSSION TITLE
Apply real data (UOW,Entity Framework)

### DISCUSSION THREAD

**Discussion by ava-vn** [LeftTwixWand/Inventory#27] [Ideas]
The project is great.
When we will apply real data? using UOW pattern and Entity Framework?

**Comment by ava-vn** [comment]
@LeftTwixWand hope we have feature soon!

**Reply by LeftTwixWand** [reply]
Hey @maihungdev, thank you so much for this discussion!🚀
There is a branch, called [database](https://github.com/LeftTwixWand/Inventory/tree/database), where I'm experimenting with the ef core for SQLite database. And in the nearest feature it will appear in master branch.

Also, @dicksonkimeu asked to add some WEB Api storage provider as an example. And it shouldn't be so hard to do, but I will definitely take some time to implement.

Anyway - feel free to create new issues or to start new discussions, to share your ideas and needs.
This project has created exactly for such purpose - to show the community how to build modern WinUI apps.

**Reply by ava-vn** [reply]
Thanks! Hopefully the features will be out soon.

**Comment by ava-vn** [comment]
Will the repository focus UI/UX?
I tried use database feature, but it's not working.
I have issue in migration data
Hope this repository release real data feature with SQL soon.

**Reply by LeftTwixWand** [reply]
Hi! This repository will cover all the aspects, needed for production apps development.
The key features are:
- Production architecture 
- Modern design
- Various data storages
- Tests
